inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/generic_eltwise_ref.cl

```diff
@@ -57,10 +57,10 @@ KERNEL(eltwise)(
     #else
         const uint d1 = get_global_id(0);
         const uint d2 = (uint)get_global_id(1) % OUTPUT_SIZES[1];
-        const uint d3 = (uint)get_global_id(1) / OUTPUT_SIZES[1] % OUTPUT_SIZE[2];
-        const uint d4 = (uint)get_global_id(1) / OUTPUT_SIZES[1] / OUTPUT_SIZE[2];
-        const uint d5 = (uint)get_global_id(2) % OUTPUT_SIZES[3];
-        const uint d6 = (uint)get_global_id(2) / OUTPUT_SIZES[3];
+        const uint d3 = (uint)get_global_id(1) / OUTPUT_SIZES[1] % OUTPUT_SIZES[2];
+        const uint d4 = (uint)get_global_id(1) / OUTPUT_SIZES[1] / OUTPUT_SIZES[2];
+        const uint d5 = (uint)get_global_id(2) % OUTPUT_SIZES[4];
+        const uint d6 = (uint)get_global_id(2) / OUTPUT_SIZES[4];
```

Comment:
Why do you use array `OUTPUT_SIZES` instead of `OUTPUT_SIZE_(X,Y,Z)`? I think the last variant is more readable.

Reply:
Refactoring can be done separately. I just fixed compilation/accuracy issues